docs/data/joy/components/autocomplete/GitHubLabel.tsx

```diff
@@ -202,7 +212,17 @@ export default function GitHubLabel() {
                     <Typography level="title-sm">{option.name}</Typography>
                     <Typography level="body-xs">{option.description}</Typography>
                   </Box>
-                  <CloseIcon sx={{ visibility: selected ? 'visible' : 'hidden' }} />
+                  <CloseIcon
+                    sx={[
+                      selected
```

Comment:
Not to us, document this change in the migration guide in regards to using Pigment CSS.